frontend/src/main/scala/bloop/engine/tasks/ScalaJsToolchain.scala

```diff
@@ -23,22 +23,23 @@ class ScalaJsToolchain private (classLoader: ClassLoader) {
    * @param project   The project to link
    * @param config    The configuration for Scalajs
    * @param mainClass The fully qualified main class name
+   * @param target    The output file path
    * @param logger    The logger to use
    * @return The absolute path to the generated JS source.
    */
   def link(
       config: JsConfig,
       project: Project,
       mainClass: String,
+      target: AbsolutePath,
       logger: Logger
-  ): Task[Try[AbsolutePath]] = {
+  ): Task[Try[Unit]] = {
```

Comment:
Can we keep the return type here too?

Reply:
I don't really feel strongly about that, but I don't like checking that's the result is a `Success(_)` and do nothing with that. If we change it, we should do the same in the `ScalaNativeToolchain` though!